src/components/views/messages/MVideoBody.tsx

```diff
@@ -29,6 +29,8 @@ interface IProps {
     mxEvent: any;
     /* called when the video has loaded */
     onHeightChanged: () => void;
+
+    scBubbleActionBar: any;
```

Comment:
This should preferably stay searchable what it would if just removing sc because things makes maintenance a whole lot easier. Is just removing sc what you mean?

Reply:
> Is just removing sc what you mean?

Yes